.github/workflows/arm64.yml

```diff
@@ -11,13 +11,16 @@ env:
   MAVEN_USERNAME: ${{ secrets.NEXUS_USERNAME }}
   TAS_ENVIRONMENT: ./tests/environment
   TAS_SCRIPTS: ../alfresco-community-repo/packaging/tests/scripts
+  JIRA_TOKEN: ${{ secrets.JIRA_TOKEN }}
```

Comment:
This could be an env var of the specific step that needs it.

Reply:
Done.
Ref. https://github.com/Alfresco/acs-packaging/pull/2450